### ISSUE TITLE
Missing plugin on Heroku hosted application, working fine locally

### ISSUE THREAD

**Issue by lucasgerard** [maxrossello/redmine_app_timesheets#79]
Hi,

The plugin is working fine and appear locally, but as I push my app on Heroku, the plugins index page is empty and there is no plugin...

I've removed public/plugin_assets - and even everything to be sure - from my .gitignore and I've also ran `heroku run rake redmine:plugins:migrate`. No success. Is there any additionnal configuration necessary to Heroku deployment?

Might be worthing mentioning that I've got the following depreciation warnings but I don't know if it comes from here:

```
DEPRECATION WARNING: You have Rails 2.3-style plugins in vendor/plugins! Support for these plugins will be removed in Rails 4.0. Move them out and bundle them in your Gemfile, or fold them in to your app as lib/myplugin/* and config/initializers/myplugin.rb. See the release notes for more on this: http://weblog.rubyonrails.org/2012/1/4/rails-3-2-0-rc2-has-been-released. (called from <top (required)> at /app/Rakefile:7)
Plugins in vendor/plugins (/app/vendor/plugins) are no longer allowed. Please, put your Redmine plugins in the `plugins` directory at the root of your Redmine directory (/app/plugins).
However, my plugins are already included in the 'plugins' directory and I'm now kind of stuck on how to solve this problem...
```

Rails v3.2.17 with Redmine v2.5.1.

Thanks for your help!

**Comment by maxrossello** [comment]
In redmine 2.x plugins must go into /plugins, not /vendor/plugins.
Let me know if this helps you.

**Comment by lucasgerard** [comment]
Hi Max, thanks for your quick reply. /vendor/plugins is empty and the plugins are in /plugins indeed.

In the meantime, I found that it might be related to git submodules problem, as the redmine_app_timesheets folder was present but figured to be empty while running `heroku run bash`.

So I deleted the two folders and reinstalled the plugin using: 

```
git submodule add https://github.com/maxrossello/redmine_app__space.git plugins/redmine_app__space
git submodule add https://github.com/maxrossello/redmine_app_timesheets.git plugins/redmine_app_timesheets
```

Plugins are now working, just not sure what configuration I might have lost deleting the local plugins.
